Lookup: Add a Getting Started tutorial (#3289)
DevExpress/devextreme-documentation: concepts/05 UI Components/Lookup/00 Getting Started with Lookup/20 Configure Search.md

Lookup allows users to search through the drop-down list items. To configure the search, set the following properties:

- [searchMode](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#searchMode): *"contains"* | *"startswith"*     
Specifies a comparison operation used to search Lookup's items. 

- [searchExpr](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#searchExpr)    
Specifies the name of a data source item field or an expression whose value is compared to the search criteria.

- [minSearchLength](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#minSearchLength)    
Specifies the minimal number of characters that must be entered into the text box to begin a search.

To disable the search, set the [searchEnabled](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#searchEnabled) property to **false**.

> In the following code, the **searchMode** property is set to *"contains"*. This mode allows users to find items that contain the search value. The **searchExpr** property is set to an array of two data fields, and users can search through both. The **minSearchValue** property indicates that the search begins only when users type in two or more characters.

Reviewer: ```suggestion
In the following code, the **searchMode** property is set to *"contains"*. This mode allows users to find items that contain the search value. The **searchExpr** property is set to an array of two data fields, and the component searches them both. The **minSearchValue** property indicates that the search begins only when users type in two or more characters.
```
